paperqa/types.py

```diff
@@ -54,9 +54,9 @@ def set_llm_session_ids(session_id: UUID):
 
 
 class Doc(Embeddable):
-    docname: str
+    docname: str | None = None
+    dockey: DocKey | None = None
```

Comment:
Why make these optional and defaulted to `None`?

I am not against it, but more just wondering why. Can you add a `description` stating what these should be and what `None` means